changelog/unreleased/issue-2942

```diff
@@ -0,0 +1,6 @@
+Bugfix: Make --exclude-larger-than handle disappearing files
+
+If files restic encounters during backup are deleted before restic has a chance to
+process them and check the size, a panic may occur. This is fixed now.
```

Comment:
I suggest changing this to the following, to make it more in the past:

~~~
There was a small bug in the backup command's --exclude-larger-than
option where files that disappeared between scanning and actually
backing them up to the repository caused a panic. This is now fixed.
~~~